### PR TITLE
Update readme for 1.3

### PR DESCRIPTION
**Pull request by DustinCampbell**
@gregg-miskelly, @caslan: If you send me information for the debugger, I'll incorporate it into this PR.

### CODE REVIEW

**Comment by DustinCampbell** [comment]
Addresses #578

**Comment by gregg-miskelly** [comment]
@DustinCampbell I was just comping you an email about this. Great minds think alike :). I will send you some text.

**Comment by DustinCampbell** [comment]
Also addresses #553

**Comment by DustinCampbell** [comment]
OK, I added the debugger what's new items. I reworded some of the text a bit, so please take a look.

**Comment by DustinCampbell** [comment]
Also, I arbitrarily ordered them to highlight some of the more important items. Let me know if you disagree with the order and I'll change it.

File: README.md [hunk 1]
```diff
@@ -9,28 +9,39 @@ Welcome to the C# extension for Visual Studio Code! This preview provides the fo
...
+* Generation of tasks.json and launch.json files can now properly handle nested projects. [#170](https://github.com/OmniSharp/omnisharp-vscode/issues/170)
+* New UI that makes it easy to select a process ID when attaching the debugger to another process. Note: If you have an existing launch.json file, you can re-generate it by deleting the file, closing your workspace in Visual Studio Code and opening it again. Or, you can open the launch.json file and change the `processId` value to `"${command.pickProcess}"`.
+* Support for debugging in .cshtml files. To enable this, add a `sourceFileMap` entry to your launch.json with the following content: `"sourceFileMap": { "/Views": "${workspaceRoot}/Views" }`
+* Support for conditional breakpoints
+* New configuration option to enable [stepping into properties and operators](https://github.com/OmniSharp/omnisharp-vscode/blob/release/debugger.md#stepping-into-properties-and-operators).
+* Duplicate warnings and errors should no longer accumulate in Unity projects [#447](https://github.com/OmniSharp/omnisharp-vscode/issues/447)
+
```
Comment (gregg-miskelly) [inline README.md]: Two other debugger items that I realized I forgot --

* Support for setting the value of variables in the debugger. To do this, right click on the variable and invoke 'Set Value'.
* Display of type names for variables changed in the debugger to support VS Code's variable assignments. A shortened form of the type names is now displayed as part of the variable name, and the full type name can be viewed by hovering over the name.

**Comment by gregg-miskelly** [comment]
Otherwise LGTM

**Comment by rajkumar42** [comment]
:+1:

**Comment by DustinCampbell** [comment]
@gregg-miskelly: Does this look right? These two items appeared to me to be related, so I collapsed them. Was that correct?

**Comment by DustinCampbell** [comment]
ping?

**Comment by gregg-miskelly** [comment]
LGTM

**Pull request merged**